app/controllers/shelters_controller.rb

```diff
@@ -11,8 +11,14 @@ def new
   end
 
   def create
-    Shelter.create(shelter_params)
-    redirect_to '/shelters'
+    shelter = Shelter.create(shelter_params)
```

Comment:
not sure if it matters, but `create` could be `new`. Calling `create` will try to save it to the database, which is already happening when you call `shelter.save` on the next line